### ISSUE TITLE
ReadTheDocs compatible docstrings 

### ISSUE THREAD

**Issue by isaacmg** [AIStream-Peelout/flow-forecast#341]
We have a lot of antiquated docstrings that don't render well into ReadTheDocs.  A kind of grunge (but incredibly useful) task would be to refactor these docstrings into proper ReadTheDocs format. This would allow us to render them effectively...

**Comment by theadammurphy** [comment]
I use VS Code and there is an [extension](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) which automatically generates docstrings in readthedocs format.

[train_transformer_style](https://flow-forecast.readthedocs.io/en/latest/pytorch_training.html) looks really good, so I assume you'd want them all in that style?

**Comment by isaacmg** [comment]
Yes if you wanted to open a PR for that it would be really helpful.
